topics.controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchAllTopics } = require("./models/topics.models");
+
+exports.getAllTopics = (req, res, next) => {
+  fetchAllTopics().then((topics) => {
+    res.status(200).send(topics);
+  });
+};
```

Comment:
Don't forget to add catch blocks to your controllers at the end of the promise chain.